packages/paste-website/src/components/assistant/LoadingMessage.tsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable camelcase */
+import { Box } from "@twilio-paste/box";
+import { ChatBubble, ChatMessage, ChatMessageMeta, ChatMessageMetaItem } from "@twilio-paste/chat-log";
+import { SkeletonLoader } from "@twilio-paste/skeleton-loader";
+import { useUID } from "@twilio-paste/uid-library";
+import * as React from "react";
+
+import { Logo } from "../../assets/Logo";
+import { useAssistantRunStore } from "../../stores/assistantRunStore";
+import { formatTimestamp } from "../../utils/formatTimestamp";
+
+const getRandomNumber = (max: number): number => {
+  return Math.floor(Math.random() * max);
+};
+
+const STATUS_MAP = {
+  queued: "Queued...",
+  in_progress: "Researching...",
+  requires_action: "Researching...",
+  cancelling: "Concelling...",
+  cancelled: "Cancelled.",
+  failed: "Failed.",
+  completed: "Finished.",
+  expired: "Expired.",
+};
+
+export const LoadingMessage: React.FC<{ maxWidth: number }> = ({ maxWidth }) => {
+  const activeRun = useAssistantRunStore((state) => state.activeRun);
+
+  const newDateTime = new Date();
+  const timestamp = Math.floor(newDateTime.getTime() / 1000);
+
+  const randomWidths = React.useMemo(() => {
```

Comment:
suggestion (non-blocking): would be cool if this was inbuild to the SkeletonLoader. Like `<SkeletonLoader randomWidth={[200, 500]} randomHeight={[100,200]} />`